content/analyze/methods/methods.md

```diff
@@ -9,3 +9,13 @@ linked:
     - ./scmap.md
     - ./stream.md
 ---
+
+##  Methods Packages
+The HCA Methods Package Registry lists tools for performing analyses involving scRNA-seq, scDNA-seq, in situ transcriptomics, multi-modal and other computational biology approaches. 
+
+These solutions are not built by the HCA DCP implementation team but are solutions we foster and support. 
+
+You can use these methods packages by following the installation and usage instructions in the method's detail page.  Portal developers can also find instructions for integrating these methods into an app on each method's detail page.
```

Comment:
Let's delete or (preferably, if possible) comment out this part about detail pages until they're ready.
```suggestion
```